jbpm-process-svg/src/main/java/org/jbpm/process/svg/SVGImageProcessor.java

```diff
@@ -62,16 +62,17 @@ public SVGProcessor getProcessor() {
     //Static methods to keep backward compatibility
 
     public static String transform(InputStream svg, List<String> completed, List<String> active) {
-        return transform(svg, completed, active, null, COMPLETED_COLOR, COMPLETED_BORDER_COLOR, ACTIVE_BORDER_COLOR);
+        return transform(svg, completed, active, null, COMPLETED_COLOR, COMPLETED_BORDER_COLOR, ACTIVE_BORDER_COLOR, "", "");
     }
 
     public static String transform(InputStream svg, List<String> completed, List<String> active, Map<String, String> subProcessLinks){
-        return transform(svg, completed, active, subProcessLinks, COMPLETED_COLOR, COMPLETED_BORDER_COLOR, ACTIVE_BORDER_COLOR);
+        return transform(svg, completed, active, subProcessLinks, COMPLETED_COLOR, COMPLETED_BORDER_COLOR, ACTIVE_BORDER_COLOR, "", "");
     }
 
     public static String transform(InputStream svg, List<String> completed, List<String> active,
                                    Map<String, String> subProcessLinks, String completedNodeColor,
-                                   String completedNodeBorderColor, String activeNodeBorderColor) {
+                                   String completedNodeBorderColor, String activeNodeBorderColor,
```

Comment:
Overload the transform method, so that both method signatures are present:
transform(InputStream svg, List<String> completed, List<String> active,
                                   Map<String, String> subProcessLinks, String completedNodeColor,
                                   Map<String, String> subProcessLinks, String completedNodeColor,
                                   String completedNodeBorderColor, String activeNodeBorderColor)
and
transform(InputStream svg, List<String> completed, List<String> active,
                                   Map<String, String> subProcessLinks, String completedNodeColor,
                                   Map<String, String> subProcessLinks, String completedNodeColor,
                                   String completedNodeBorderColor, String activeNodeBorderColor, String svgWidth, String svgHeight)
.
Then get rid of transform(....., "", "") invocations in other locations than in this class.

Reply:
Also I'd argue for passing nulls rather then emptystrings (as we're not passing the values anyway)